docs/guides/connecting_to_your_data/cloud/gcs/spark.md

```diff
@@ -2,4 +2,153 @@
+
+```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_python_example.py#L53
+```
+
+</TabItem>
+</Tabs>
+
+If you specified a GCS path containing CSV files you will see them listed as `Available data_asset_names` in the output of `test_yaml_config()`.
+
+Feel free to adjust your configuration and re-run `test_yaml_config()` as needed.
+
+### 4. Save the Datasource configuration to your DataContext
+
+Save the configuration into your `DataContext` by using the `add_datasource()` function.
+
+<Tabs
+  groupId="yaml-or-python"
+  defaultValue='yaml'
+  values={[
+  {label: 'YAML', value:'yaml'},
+  {label: 'Python', value:'python'},
+  ]}>
+  <TabItem value="yaml">
+
+```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_yaml_example.py#L54
+```
+
+</TabItem>
+<TabItem value="python">
+
+```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_python_example.py#L55
+```
+
+</TabItem>
+</Tabs>
+
+### 5. Test your new Datasource
+
+Verify your new Datasource by loading data from it into a `Validator` using a `BatchRequest`.
+
+<Tabs
+  defaultValue='runtime_batch_request'
+  values={[
+  {label: 'Specify a GCS path to single CSV', value:'runtime_batch_request'},
+  {label: 'Specify a data_asset_name', value:'batch_request'},
+  ]}>
+  <TabItem value="runtime_batch_request">
+
+Add the GCS path to your CSV in the `path` key under `runtime_parameters` in your `BatchRequest`.
+
+```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_yaml_example.py#L57-L63
+```
+Then load data into the `Validator`.
+```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_yaml_example.py#L71-L77
+```
+
+  </TabItem>
+  <TabItem value="batch_request">
+
+Add the name of the data asset to the `data_asset_name` in your `BatchRequest`.
+
+```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_yaml_example.py#L83-L88
+```
+Then load data into the `Validator`.
+```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_yaml_example.py#L96-L102
+```
+
+  </TabItem>
+</Tabs>
+
+
+<Congratulations />
+
+## Additional Notes
+
+<SparkAdditionalNotes />
+
+To view the full scripts used in this page, see them on GitHub:
+
+- [spark_s3_yaml_example.py](https://github.com/great-expectations/great_expectations/blob/develop/tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_yaml_example.py)
+- [spark_s3_python_example.py](https://github.com/great-expectations/great_expectations/blob/develop/tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/spark/inferred_and_runtime_python_example.py)
```

Comment:
These should link to the gcs example docs

Reply:
Great catch -- fixed!